packages/core/src/mm/actors/sHammerVtx.vtx.inc

```diff
@@ -0,0 +1,208 @@
+	VTX(486, 364, -100, 71, 2, 79, 0, 0, 255),
```

Comment:
This shouldn't be necessary, we already extract the hammer into it's own object file for child link OoT (tho it uses segment `A`), we should reuse that.

Reply:
Altho not urgent, I can handle that later.